tests/test_install.sh

```diff
@@ -17,50 +17,32 @@ rm -rf "$working_dir"
 mkdir -p "$working_dir"
 cd "$working_dir"
 
+# Start ephemeral network in the background
+network_live_time=60
+timeout --signal=SIGINT --kill-after=${network_live_time}s --preserve-status ${network_live_time}s \
+"$INSTALL_PREFIX"/bin/sandbox.sh --verbose &
+
+# Wait for service to be open
+sleep 45
+
+# # Issue tutorial transactions to ephemeral network
 python3.8 -m venv env
 source env/bin/activate
-python -m pip install -U -r "$INSTALL_PREFIX"/bin/requirements.txt
 python -m pip install ../../../python
+python ../../../python/tutorial.py ./workspace/sandbox_0/0.ledger/ ./workspace/sandbox_common/
```

Comment:
Maybe the waiting ought to happen at the top of tutorial, using the client timeout logic we have in Python anyway?

Reply:
As discussed earlier, we wait for service opening directly inside `test_install.sh` (where the `sleep ...` was). I believe this is neater and keeps the `tutorial.py` untouched.